package.json

```diff
@@ -14,35 +14,57 @@
   "private": true,
   "dependencies": {
     "@apollo/client": "^3.5.8",
+    "@blueprintjs/core": "^3.47.0",
     "@blueprintjs/icons": "^3.32.0",
+    "@blueprintjs/select": "^3.16.6",
+    "@craco/craco": "^6.2.0",
     "@nrwl/next": "13.8.1",
     "@radix-ui/react-dialog": "^0.1.5",
     "@radix-ui/react-tabs": "^0.1.5",
     "@radix-ui/react-tooltip": "^0.1.7",
     "@sentry/nextjs": "^6.19.3",
     "@sentry/react": "^6.19.2",
     "@sentry/tracing": "^6.19.2",
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "@types/faker": "^5.5.8",
+    "@types/jest": "^26.0.15",
     "@types/lodash": "^4.14.180",
+    "@types/node": "^12.0.0",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
+    "@types/react-router-dom": "^5.1.8",
     "@types/uuid": "^8.3.4",
     "@vegaprotocol/smart-contracts-sdk": "^1.6.0",
     "@vegaprotocol/vegawallet-service-api-client": "^0.4.6",
     "@walletconnect/ethereum-provider": "^1.7.5",
+    "@walletconnect/web3-provider": "^1.6.6",
     "@web3-react/core": "8.0.20-beta.0",
+    "@web3-react/injected-connector": "^6.0.7",
```

Comment:
chore (blocking): can we tidy up package.json 

I believe we can remove the following:
@walletconnect/web3-provider
@web3-react/injected-connector
@web3-react/network-connector
@web3-react/walletconnect-connector
eslint-plugin-wdio
bn.js
@craco/craco
@testing-library/user-event

Also the following are duplicated in deps/dev deps. Can we a) take the later version of the two, b) ensure its in dev deps:
- @types/lodash
- @types/node
- @types/jest
- @types/react
- @types/react-router-dom
- @types/react-dom
- @testing-library/react
- @testing-library/jest-dom
- typescript

Reply:
Only @web3-react/injected-connector was required

Reply:
@mattrussell36 @types/jest-dom isn't currently used. Did you mean "@testing-library/jest-dom": "^5.16.2"?